client/src/components/TripCard.js

```diff
@@ -18,21 +18,23 @@ class TripCard extends Component {
         {!trip.id && "Loading trip"}
         {trip.id && (
           <>
-            <div className="card">
-              <div className="card-image">
-                <img
-                  src="https://staticmapmaker.com/img/google.png"
-                  alt="Google Map of Albany, NY"
-                />
-              </div>
-              <div className="card-content">
-                <div>{trip.name}</div>
-                <div>Start: {trip.start}</div>
-                <div>End: {trip.end}</div>
-                <CardButton onClick={this.handleSingleTrip(trip.id)}>
-                  >
-                </CardButton>
-              </div>
+            <div key={trip.id} trip={trip} className="card">
```

Comment:
Can we remove `key={trip.id} trip={trip}` from this line? `div` isn't a react component so it shouldn't be there